src/ranking.jl

```diff
@@ -171,3 +171,165 @@ Missing values are assigned rank `missing`.
 """
 tiedrank(x::AbstractArray; sortkwargs...) =
     _rank(_tiedrank!, x, Float64; sortkwargs...)
+
+"""
+    quantilerank(itr, value; method=:inc)
+
+Compute the quantile(s)-position in [0, 1] interval of a `value` relative to a collection 
+`itr`, e.g. a quantile rank of x means that (x*100)% of the elements in `itr` are less than 
+(or less than or equal) the given `value`, according to the respective method used. 
```

Comment:
Given the discussion I would propose to just do:
```suggestion
Compute the quantile(s)-position in [0, 1] interval of a `value` relative to a collection 
`itr`. 
```

Reply:
Done in commit 6bbfa5d.